src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,324 @@
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println("    ☹ OOPS!!! The description of a delete cannot be empty! Please try again.");
+                    System.out.println(SPLIT_LINE);
+                }
+            } else {
+                if (order.contains("todo")) {
+                    try {
+                        String task = order.substring(order.indexOf("todo") + FIND_TASK_TODO);
+                        list.add(new Todo(task));
+                    } catch (IndexOutOfBoundsException e) {
+                        System.out.println(SPLIT_LINE);
+                        System.out.println("    ☹ OOPS!!! The description of a todo cannot be empty.");
+                        System.out.println(SPLIT_LINE);
+                        continue;
+                    }
+                } else if (order.contains("event")) {
+                    try {
+                        String task = order.substring(order.indexOf("event")
+                            + FIND_TASK_EVENT, order.indexOf("/") - 1);
+                        String at = order.substring(order.indexOf("/at") + FIND_TIME);
+                        System.out.println("hello" + at);
+                        list.add(new Event(task, at));
+                    } catch (IndexOutOfBoundsException e) {
+                        System.out.println(SPLIT_LINE);
+                        System.out.println("    ☹ OOPS!!! The description of a event cannot be empty.");
+                        System.out.println(SPLIT_LINE);
+                        continue;
+                    }
+                } else if (order.contains("deadline")) {
+                    try {
+                        String task = order.substring(order.indexOf("deadline")
+                            + FIND_TASK_DEADLINE, order.indexOf("/") - 1);
+                        String by = order.substring(order.indexOf("/by") + FIND_TIME);
+                        list.add(new Deadline(task, by));
+                    } catch (IndexOutOfBoundsException e) {
+                        System.out.println(SPLIT_LINE);
+                        System.out.println("    ☹ OOPS!!! The description of a deadline cannot be empty.");
+                        System.out.println(SPLIT_LINE);
+                        continue;
+                    }
+                } else {
+                    try {
+                        invalidInput(true);
+                    } catch (DukeException e) {
+                        System.out.println(SPLIT_LINE);
+                        System.out.println("    ☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
+                        System.out.println(SPLIT_LINE);
+                        continue;
+                    }
+                }
+                System.out.println(SPLIT_LINE);
+                System.out.println("    Got it. I've added this task:");
+                System.out.println("     " + list.get(list.size() - 1));
```

Comment:
Avoid having such nested logic in print statements as the purpose of print statements is only to print and not do anything else. Seperate this out into separate lines